hibp.js

```diff
@@ -126,23 +120,27 @@ const HIBP = {
       }
     }
 
-    if (includeUnsafe) {
-      return foundBreaches;
+    if (includeSensitive) {
+      return this.filterBreaches(foundBreaches);
     }
-    return this.filterOutUnsafeBreaches(foundBreaches);
+    return this.filterBreaches(foundBreaches).filter(
```

Comment:
Both of these branches call `this.filterBreaches()`. Not sure if it makes sense to move this up around L118 (where we populate `foundBreaches` initially), but putting it all in a single command is probably a bit too aggressive:

```js
foundBreaches = this.filterBreaches(allBreaches.filter(breach => breachedAccount.websites.includes(breach.Name)));

// vs

foundBreaches = allBreaches.filter(breach => breachedAccount.websites.includes(breach.Name));
foundBreaches = this.filterBreaches(foundBreaches);
```

Which would reduce this code to something like:

```js
if (includeSensitive) {
  return foundBreaches;
}
return foundBreaches.filter(breach => !breach.IsSensitive);
```

I guess the alternative would be to pass the `includeSensitive` argument to the `this.filterBreaches()` method and do the additional check there, which could reduce this `if..else` code to the following:

```js
return this.filterBreaches(foundBreaches, includeSensitive);
```

But all these are pretty nano-nits, and pointless code juggling. Feel free to merge whenever.